graphql-schemas/schema.yml

```diff
@@ -1212,14 +1212,14 @@ confs:
   - { name: provider, type: string, isRequired: true }
   - { name: enforced, type: boolean, isRequired: true }
 
-- name: FleetLabels_v1
-  datafile: /openshift/fleet-labels-1.yml
+- name: FleetLabelsSpec_v1
+  datafile: /openshift/fleet-labels-spec-1.yml
   fields:
   - { name: schema, type: string, isRequired: true }
   - { name: path, type: string, isRequired: true }
   - { name: name, type: string, isRequired: true, isUnique: true, isSearchable: true }
   - { name: description, type: string }
-  - { name: ocms, type: OpenShiftClusterManager_v1, isList: true, isRequired: true }
+  - { name: ocm, type: OpenShiftClusterManager_v1, isRequired: true }
```

Comment:
this is a breaking change, will it cause error in integration before schema sync in qontract-reconcile?

Reply:
It wont cause error - no integration in prod using this schema

Reply:
In general: no change in this PR is used by any integration that is running right now

Reply:
all good then!